network/struct.go

```diff
@@ -107,6 +109,7 @@ func NewServerIdentity(public abstract.Point, address Address) *ServerIdentity {
 		Public:  public,
 		Address: address,
 		ID:      ServerIdentityID(uuid.NewV5(uuid.NamespaceURL, url)),
+		Cert:    []byte{},
```

Comment:
"You have heard: bug for bug, nitpick for nitpick, but I say: do not resist commenters. If they comment on a bad line, show them your worst..."

Anyway - added a `NewTLSServerIdentity`, even though it is used twice for the moment.